src/apis/AppState/index.js

```diff
@@ -2,7 +2,7 @@ import ExecutionEnvironment from 'fbjs/lib/ExecutionEnvironment';
 import findIndex from 'array-find-index';
 import invariant from 'fbjs/lib/invariant';
 
-const EVENT_TYPES = [ 'change' ];
+const EVENT_TYPES = ['change'];
```

Comment:
¯\_(ツ)_/¯ Something like prettier is so hard because everyone has ingrained habits like that, and they all conflict. I can't find the issue where we talked about it, but it was a complaint. I have no idea what to do except... say sorry? :/

I mean, we *could* make the bracketSpacing option more configurable but we really, really don't want to add many more configuration options unless we really need to. 

If you want to continue discussing it, please file an issue! I think, even if we've talked about it before, it's worth talking with newcomers because it helps them understand how we got there and hopefully makes them feel better about it. And maybe you could even change our minds!